test/lib/types/library/LibraryFormatter.js

```diff
@@ -284,6 +284,21 @@ test("getDotLibrary: reads correctly", async (t) => {
 	t.deepEqual(fsPath, expectedPath, ".library fsPath is correct");
 });
 
+test("getDotLibrary: reads correctly call again", async (t) => {
```

Comment:
What is this testing exactly? The caching mechanism inside `getDotLibrary`?

Reply:
Yes

Reply:
But if I remove the caching here:
https://github.com/SAP/ui5-builder/blob/7773b4e8c6154330a16e6c7a5aff79ecc5579c63/lib/types/library/LibraryFormatter.js#L243-L245

The test will still be successful because it doesn't actually test whether something got cached (e.g. no second glob call).

---

Also, there is already tests for this. It just seem to be incorrect since its missing a second call: https://github.com/SAP/ui5-builder/blob/7773b4e8c6154330a16e6c7a5aff79ecc5579c63/test/lib/types/library/LibraryFormatter.js#L344

Reply:
exactly

Reply:
> exactly

...

So we gonna fix the existing test instead?

Reply:
👍